baremetal_benchmarking/gramine_libs.py

```diff
@@ -197,6 +197,7 @@ def update_manifest_file(test_config_dict):
 def generate_sgx_token_and_sig(test_config_dict):
     sgx_exec = len(list(e_mode for e_mode in test_config_dict['exec_mode'] if 'gramine-sgx' in e_mode))
     if sgx_exec > 0:
+        print("\n-- Generating SGX manifest and token file..")
         sign_cmd = "gramine-sgx-sign --manifest {0}.manifest --output {0}.manifest.sgx".format(test_config_dict['manifest_name'])
         token_cmd = "gramine-sgx-get-token --output {0}.token --sig {0}.sig".format(test_config_dict['manifest_name'])
```

Comment:
Not something for this PR, but we now don't need to call gramine-sgx-get-token explicitly, it gets called automatically by Gramine. In fact this will throw an error in the near future.